Islandora-CLAW/controlled_access_terms
# Replace field_geo_broader with taxonomy-provided parent field

**GitHub Issue**: https://github.com/Islandora-CLAW/CLAW/issues/1059

# What does this Pull Request do?

Replace field_geo_broader with taxonomy-provided parent field

# What's new?

* Removes field_geo_broader
* Updates vocabulary to use hierarchy and remove references to field_geo_broader
* Updates RDF mapping to use parent field for schema:containedInPlace

# How should this be tested?

- Apply the PR
- Create Geographic Locations
- Order them in a hierarchy
- Checkout the jsonld to ensure that the schema:containedInPlace is referencing the correct parent.

# Interested parties
@Islandora-CLAW/committers
